src/components/Image/index.jsx

```diff
@@ -0,0 +1,5 @@
+const Image = ({ url }) => {
+  return <img style={{ height: "500px", padding: "10px" }} src={url} alt="background" />;
```

Comment:
you can use `styled-components` for `img` tag

Reply:
done )